bin/notify-on-exclude-change

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -euo pipefail
+
+: "${SLACK_TOKEN:?The SLACK_TOKEN environment variable is required.}"
+
+bin="$(dirname "$0")"
+
+src="${1:?A source exclude txt file is required as the first argument.}"
+dst="${2:?A destination exclude txt s3:// URL is required as the second argument.}"
+
+dst_local="$(mktemp -t exclude-XXXXXX.txt)"
+
+diff="$(mktemp -t exclude-additions-XXXXXX)"
+
+trap "rm -f '$dst_local' '$diff'" EXIT
+
+aws s3 cp --no-progress "$dst" - | gunzip -cfq > "$dst_local"
+
+comm -13 \
+    <(sort "$dst_local") \
+    <(sort "$src") \
+    > "$diff"
+
+if [[ -n "$(< "$diff")" ]]; then
```

Comment:
A more idiomatic test that doesn't require reading in the file contents is:

```sh
if [[ -s "$diff" ]]; then
```

see <https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html>.

Reply:
Thanks. I also changed a similar conditional in `notify-on-metadata-change` in the latest commit.

Reply:
Ah, in this case, the use of `-n` in `notify-on-metadata-change` was intentional, see https://github.com/nextstrain/ncov-ingest/commit/2e7093fb58894bb028b96450bc1e1ac87ce6d0af. I should have left a comment in the program as well.

Reply:
Gotcha. I documented that in a comment in the fixed up version of the latest commit